python/tk_desktop/desktop_window.py

```diff
@@ -722,6 +725,13 @@ def __launch_app_proxy_for_project(self, project, pipeline_configuration_id=None
             else:
                 raise RuntimeError("unknown platform: %s" % sys.platform)
 
+            if config_path is None:
+                raise RuntimeError("No path set for %s on the Pipeline "
+                                   "Configuration id %d. Please make sure that "
+                                   "it is set on your Shotgun website." %
```

Comment:
Something like this?

![screen shot 2015-10-29 at 11 10 17 am](https://cloud.githubusercontent.com/assets/1843555/10822476/ce2fa92a-7e2d-11e5-9e60-69111e82c956.png)